### DISCUSSION TITLE
Connection abbort and incorrect download of files with Fritz!box Wireguard VPN

### DISCUSSION THREAD

**Discussion by hirschferkel** [iterate-ch/cyberduck#15631] [Q&A]
I guess this issue is caused by Wireguard VPN. 

I use two Fritz!boxes which are connected with wireguard VPN. When I try to download bigger files with FTP-Client, it will constantly abort the connection and if downloads get finished, most of these are broken ( ...cyberducksegement instead of e.g. .gz or zip file).

Even canceling the download will make Cyberduck crash instantly.

Any idea how to fix it?

Best
<img width="839" alt="Bildschirmfoto 2024-02-12 um 11 31 59" src="https://github.com/iterate-ch/cyberduck/assets/16016422/bf5c2fb0-3325-4fdc-a961-0baad355708a">
<img width="692" alt="Bildschirmfoto 2024-02-12 um 11 32 28" src="https://github.com/iterate-ch/cyberduck/assets/16016422/f63366a7-1ae9-40ea-9b9f-27691fa6d11f">
<img width="394" alt="Bildschirmfoto 2024-02-12 um 11 36 00" src="https://github.com/iterate-ch/cyberduck/assets/16016422/e8d98afc-53b7-497a-ae6d-4bb03b128923">

**Comment by hirschferkel** [comment]
Maybe you did not understand the issue. Everything works perfect with the VPN, I only suffer these abortions with Cyberduck. Anyway I do not know if it is caused by VPN or not, but it's just a strong estimate. Thought you should know about it, but as I see it does not meet your interest. Best

**Comment by tatze96** [comment]
Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), recreate the issue, and send us the log file afterward? Also, please send us the [crash report](https://docs.cyberduck.io/cyberduck/support/#crash-reports) of one of the mentioned crashes.

If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-15631 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.

**Comment by hirschferkel** [comment]
@tatze96 I've sent a mail to support@cyberduck.io

**Comment by tatze96** [comment]
Finding: Segmented downloads currently don't work for FTPS; see [^1].

[^1]: #15679
